pkg/build/api/types.go

```diff
@@ -142,6 +146,9 @@ const (
 
 	// BuildPhaseCancelled indicates that a running/pending build was stopped from executing.
 	BuildPhaseCancelled BuildPhase = "Cancelled"
+
+	// BuildPhaseSkipped indicates that a queued build was skipped.
+	BuildPhaseSkipped BuildPhase = "Skipped"
```

Comment:
@smarterclayton @bparees I think we should add new phase where (this is will termination phase) to indicate that this build was not cancelled but skipped (because user wiped out the queue or marked it to skip?)